aspnet/tutorials/nano-server.rst

```diff
@@ -47,31 +51,47 @@ Once you have added your Nano Server instance to your ``TrustedHosts``, you can
   $s = New-PSSession -ComputerName $ip -Credential ~\Administrator
   Enter-PSSession $s
 
-If you have successfully connected then your prompt will look like this ``[10.83.181.14]: PS C:\Users\Administrator\Documents>``
+A successful connection results in a prompt with the following format: ``[10.83.181.14]: PS C:\Users\Administrator\Documents>``
+
+Installing IIS
+--------------
+
+Add the ``NanoServerPackage`` provider from the PowerShell gallery. Once the provider is installed and imported, you can install Windows packages.
+
+.. code:: ps1
+
+  Install-PackageProvider NanoServerPackage
+  Import-PackageProvider NanoServerPackage
+  Install-NanoServerPackage -Name Microsoft-NanoServer-IIS-Package
+
 
-Installing the HttpPlatformHandler Module
------------------------------------------
+Installing the ASP.NET Core Module
+----------------------------------
 
-The :ref:`HttpPlatformHandler <http-platformhandler>` is an IIS 7.5+ module which is responsible for process management of HTTP listeners and to proxy requests to processes that it manages. At the moment, the process to install the HttpPlatformHandler Module for IIS is manual. You will need to install the latest 64-bit version of the `HttpPlatformHandler <http://www.iis.net/downloads/microsoft/HttpPlatformHandler>`_ on a regular (not Nano) machine. After installing you will need to copy the following files:
+The :ref:`ASP.NET Core Module <http-platformhandler>` is an IIS 7.5+ module which is responsible for process management of ASP.NET Core HTTP listeners and to proxy requests to processes that it manages. Currently, installing ASP.NET Core Module requires manual steps.
 
-* *%windir%\\System32\\inetsrv\\HttpPlatformHandler.dll*
-* *%windir%\\System32\\inetsrv\\config\\schema\\httpplatform_schema.xml*
+1. Install the target version of .NET Core Windows Server Hosting bundle <http://go.microsoft.com/fwlink/?LinkId=798480>__ on a regular (non-Nano) machine that has IIS enabled.
+
+2. Copy the following files from the non-Nano machine to the Nano machine:
```

Comment:
@shirhatti I have the PS script that the fundamentals team shared with me. We could use it to reduce these manual steps?

```
copy <update-this>\aspnetcore_schema.xml C:\windows\system32\inetsrv\config\schema
copy <update-this>\aspnetcore.dll C:\windows\system32\inetsrv

# Backup existing applicationHost.config
copy C:\Windows\System32\inetsrv\config\applicationHost.config C:\Windows\System32\inetsrv\config\applicationHost_BeforeInstallingANCM.config

Import-Module IISAdministration

# Initialize variables
$aspNetCoreHandlerFilePath="C:\windows\system32\inetsrv\aspnetcore.dll"
Reset-IISServerManager -confirm:$false
$sm = Get-IISServerManager

# Add AppSettings section 
$sm.GetApplicationHostConfiguration().RootSectionGroup.Sections.Add("appSettings")

# Set Allow for handlers section
$appHostconfig = $sm.GetApplicationHostConfiguration()
$section = $appHostconfig.GetSection("system.webServer/handlers")
$section.OverrideMode="Allow"

# Add aspNetCore section to system.webServer
$sectionaspNetCore = $appHostConfig.RootSectionGroup.SectionGroups["system.webServer"].Sections.Add("aspNetCore")
$sectionaspNetCore.OverrideModeDefault = "Allow"
$sm.CommitChanges()

# Configure globalModule
Reset-IISServerManager -confirm:$false
$globalModules = Get-IISConfigSection "system.webServer/globalModules" | Get-IISConfigCollection
New-IISConfigCollectionElement $globalModules -ConfigAttribute @{"name"="AspNetCoreModule";"image"=$aspNetCoreHandlerFilePath}

# Configure module
$modules = Get-IISConfigSection "system.webServer/modules" | Get-IISConfigCollection
New-IISConfigCollectionElement $modules -ConfigAttribute @{"name"="AspNetCoreModule"}

# Backup existing applicationHost.config
copy C:\Windows\System32\inetsrv\config\applicationHost.config C:\Windows\System32\inetsrv\config\applicationHost_AfterInstallingANCM.config
```